docs/guides/advanced/deepkit-type.md

```diff
@@ -0,0 +1,96 @@
+---
+id: using-with-deepkit-type
+title: 'Using with @deepkit/type'
+---
+
+This guide shows how to use `typegoose` with `@deepkit/type`.
+
+## Implementation
+
+Suppose you have this `Account` class decorated with `@deepkit/type`:
+
+```ts
+import { classToPlain, t, plainToClass } from '@deepkit/type';
+import { getModelForClass, prop } from '../../src/typegoose';
+
+enum Group {
+  confidential = 'confidential',
+  public = 'public',
+}
+
+class Account {
+  @t.mongoId.group(Group.public)
+  public _id: string;
```

Comment:
are you sure its `string` here? without `@prop` the actual type would be a `mongoose.Types.ObjectId`

(same as in the test)

Reply:
See my comment above. 

Scott

Reply:
what comment above? do you mean https://github.com/typegoose/typegoose/pull/602#discussion_r701271483, when yes - how does it relate to this being a `string` instead of `ObjectId`?

Reply:
This comment. https://github.com/typegoose/typegoose/pull/602#discussion_r701597540

Scott